### ISSUE TITLE
Error when package is not a direct child of project

### ISSUE THREAD

**Issue by acao** [coverage-report/clover-json#2]
in the clover report i generate from istanbul (nyc), we have output that looks like this:

```
<coverage generated="1545840739075" clover="3.2.0">
  <project timestamp="1545840739075" name="All files">
    <metrics statements="556" coveredstatements="482" conditionals="164" coveredconditionals="89" methods="84" coveredmethods="84" elements="804" coveredelements="655" complexity="0" loc="556" ncloc="556" packages="3" files="7" classes="7">
      <package name="test-setup">
```
so when i use the parser, i get `Cannot read property 'forEach' of undefined` on `clover-json/src/index.js:11:31`, which seems to be caused by this case. is this an issue with `istanbul-reports` clover report generator, or an issue with this parser?

**Comment by tobilg** [comment]
Can you please add some details on how you generated the report (the command)?

**Comment by acao** [comment]
@tobilg i generated it using `nyc --reporter cobertura mocha`

**Comment by tobilg** [comment]
Thanks! Could you eventually upload an example report?

**Comment by acao** [comment]
let me get back to you about that, ill need to generate a test project, but here is the library that generates the reports:
https://github.com/istanbuljs/istanbuljs/blob/master/packages/istanbul-reports/lib/clover/index.js

**Comment by tobilg** [comment]
Hm, what I don't really seemt to understand is that you're using the `cobertura` reporter, but try to parse it with `clover-json`. Why not use [cobertura-json](https://github.com/coverage-report/cobertura-json)?

Or maybe I misunderstood something? Thanks!

**Comment by ryanluker** [comment]
@tobilg there seems to be a similar issue from a user in my repo https://github.com/ryanluker/vscode-coverage-gutters/issues/204 they provided an example project that may assist you in debugging your side.
Let me know if you want me to create another ticket if this isnt the same issue.

**Comment by tobilg** [comment]
@ryanluker Thanks for reporting. I‘ll have a Look and keep you posted

**Comment by tobilg** [comment]
@ryanluker I just released version `0.3.0`. Please check and close the issue if this fixes your problem. Thanks

**Comment by ryanluker** [comment]
@tobilg thanks for looking into this so quickly I will update to the latest in my repo and test the fix :smile:

**Comment by ryanluker** [comment]
@tobilg just to wrap up this thread, I confirmed locally myself that the bug is fixed with 0.3.0 using the example project the user submitted with the bug in my repo. Thanks again!

**Comment by tobilg** [comment]
Thanks for the feedback, glad that it works for you!
